### DISCUSSION TITLE
导入到 Zotero（利用简悦增强 Zotero 的剪藏能力）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#5927] [使用技巧]
# 前因

很多学生朋友或者科研工作者都使用 Zotero 管理文献以及使用 Zotero Connector 剪藏，尤其是 7 版本出了后，可以在  Zotero 中查看和编辑 Web 了。

但 Zotero Connector 在剪藏方面做的并不好。

# 简悦的解决方案

很多用户都使用简悦对正文进行精简和优化，但发现直接使用 Zotero Connector 是「没法」剪藏简悦优化后的页面，会出现白页。

其实可以很简单的解决这个问题，即：

> 利用简悦插件 ·  Live Editor 的 **复制到原文** 功能，即可完美的将简悦优化过的页面通过 Zotero Connector 剪藏到 Zotero。

# 视频

https://github.com/Kenshin/simpread/assets/81074/82e3be52-11d2-4cd0-ab10-994fe8a1ea60

# 使用前提

务必要开启下面的选项。

<img width="475" alt="image" src="https://github.com/Kenshin/simpread/assets/81074/762ea072-873c-45b4-9808-5db248723022">

# 附录：Zotero 用户看过来

如果你是 Zotero 用户的话，仅需以下几个步骤：

1. 从 [Chrome Webstore](https://chrome.google.com/webstore/detail/simpread-reader-view/ijllcpnolfcooahcekpamkbidhejabll) 或 [Edge](https://microsoftedge.microsoft.com/addons/detail/%E7%AE%80%E6%82%A6-simpread/clgdhlhfiocphghdkdbgdlmfaafccfmc?hl=zh-CN) 下载并安装简悦

2. 安装 Live Editor  https://simpread.ksria.cn/plugins/details/y8Mai5IBwN  

3. 重启浏览器后随便进入一个页面 e.g. https://sspai.com/post/69972

4. 进入阅读模式，右下角 → 动作 → 插件触发器 → Live Editor 选项，开启上图所示的选项

5. 刷新页面（最好重启浏览器），然后再次进入这个页面，就会发现页面正下方有 Live Editor 工具条。

# 引申

使用此方式，只要是能进入阅读模式（的内容），就能导入到 Zotero。无论是知乎公式、还是推文页、小红书等均没问题。

以下是一些阅读模式 和 Live Editor 的进阶用法

- [阅读模式常用技巧汇总：重新高亮、手动模式、删除任意元素、Live Editor 等](https://github.com/Kenshin/simpread/discussions/3340)

- [Live Editor 更多用法](https://github.com/Kenshin/simpread/discussions?discussions_q=label%3A%22live+editor%22)

- [适配小红书、即刻、完美支持知乎公式高阶阅读模式](https://github.com/Kenshin/simpread/discussions?discussions_q=label%3A%22read+mode%22)

# 安装与升级

- 没有【使用同步助手 · 自动同步】的话，可通过 [此链接](https://simpread.ksria.cn/plugins/details/y8Mai5IBwN) 安装。

- 已经【使用同步助手 · 自动同步】的话，请使用 [插件管理器](https://www.yuque.com/kenshin/simpread/fcw60u#ymS1C) 安装。

**Comment by Diplproveably** [comment]
哇！感谢，现在可以剪藏到zotero了

**Reply by Kenshin** [reply]
😂 好久之前就可以了~
